active/af3-resynthesize-phi.py

```diff
@@ -225,14 +237,18 @@ def _get_patient_data_from_temp(blobid, hdcpupdatedate, patientid):
     pt_select_stmt = ("SELECT HDCOrcaBlobId, HDCPUpdateDate, HDCPersonId, FirstName, MiddleName, LastName"
                       " FROM {table}"
                       " WHERE HDCOrcaBlobId = %s AND HDCPUpdateDate = %s AND HDCPersonId = %s".format(table=common_variables.TEMP_PERSON))
-    ret = (common_hooks.ANNOTATIONS_DB.get_first(pt_select_stmt, parameters=(blobid, hdcpupdatedate, patientid))
+    try:
+      ret = (common_hooks.ANNOTATIONS_DB.get_first(pt_select_stmt, parameters=(blobid, hdcpupdatedate, patientid))
             or (None, None, None, None, None, None))
-    if ret[0] is None:
-        print(
-            "No Real Patient Name Data found in Temp DB for blobID: "
-            "{blobid}, hdcpupdatedate: {date}, patientId: {patientid}".format(
-                blobid=blobid, date=hdcpupdatedate, patientid=patientid))
-    return ret
+      if ret[0] is None:
+          print(
+              "No Real Patient Name Data found in Temp DB for blobID: "
+              "{blobid}, hdcpupdatedate: {date}, patientId: {patientid}".format(
+                  blobid=blobid, date=hdcpupdatedate, patientid=patientid))
+      return ret
+    except OperationalError as e:
```

Comment:
move the except statement up to above line 243 please

Reply:
roger.